meta-ostro/classes/supported-recipes.bbclass

```diff
@@ -270,8 +364,14 @@ python supported_recipes_eventhandler() {
             collections = supported_recipes.find_collections(pn)
             return ' (would be supported in %s)' % ' '.join(collections) if collections else ''
 
+        def re_escape(str):
+            # In contrast to re.escape(), we only escape + (as in gtk+3).
+            # Escaping all non-alphanumerics makes many entries (like linux-yocto)
+            # unnecessarily less readable (linux\-yocto).
+            return ''.join(['\\+' if x == '+' else x for x in str])
+
```

Comment:
re.sub("\+","\\+", str) would work for your case instead of creating own function.
Also, "str" in python is builtin keyword. overriding it is not good.